bundle/deploy/terraform/convert_test.go

```diff
@@ -279,3 +280,51 @@ func TestConvertExperimentPermissions(t *testing.T) {
 	assert.Equal(t, "CAN_READ", p.PermissionLevel)
 
 }
+
+func TestConvertModelServing(t *testing.T) {
+	var src = resources.ModelServing{
+		CreateServingEndpoint: &serving.CreateServingEndpoint{
+			Name: "name",
```

Comment:
Please add some additional fields here like `served_models` and `traffic_config` and assert they are correctly converted to the terraform type.

eg: `TestConvertPipeline` or `TestConvertModel`